### ISSUE TITLE
No sound output

### ISSUE THREAD

**Issue by JonnyGators** [LibreELEC/LibreELEC.tv#9182]
Describe the bug
When playing back music, there is no sound output the 3.5 mm jack on a Raspberry Pi 3

How to reproduce
Setup on Pi 3
Try to play music
Hear the silence

Information
LibreELEC Version: 12.0.0
Hardware Platform: Raspberry Pi 3
Log file
Context

**Comment by chewitt** [comment]
This is a configuration issue requiring 30 seconds of effort and since analogue output is not used by 99%+ of installs the omission of low-quality analogue output is intentional and thus not a bug. Ergo this ticket is not valid and is being closed (again). I can now see you've been told the same in forum (several hours ago) too. If you're going to waste staff time with some kind of trying-to-score-points crusade through repeated opening of invalid tickets i'm going to end that by adding you to the repo block list. Don't be lazy, just make the config change and use the low-fi audio output. If you want higher quality output get a cheap HAT DAC .. although that's going to require another 30 seconds of effort to enable the required device-tree changes to make it work and load drivers. 

TL/DR: Don't use devices that are intentionally designed to require a little tinkering if you want an experience that doesn't require a little tinkering ¯\_(ツ)_/¯
